lib/cli.ex

```diff
@@ -0,0 +1,196 @@
+defmodule Gradient.CLI do
+  @moduledoc """
+  Gradient CLI module responsible for
+  accepting shell params and starting gradualizer.
+
+  ## Command-line options
+
+    * `--no-compile` - do not compile even if needed
+    * `--no-ex-check` - do not perform checks specyfic for Elixir
+      (from ElixirChecker module)
+    * `--no-gradualizer-check` - do not perform the Gradualizer checks
+    * `--no-specify` - do not specify missing lines in AST what can
+      result in less precise error messages
+    * `--code-path` -  provide a path to the .ex file containing code for analyzed .beam
+
+    * `--no-deps` - do not import dependencies to the Gradualizer
+    * `--stop_on_first_error` - stop type checking at the first error
+    * `--infer` - infer type information from literals and other language
+      constructs,
+    * `--verbose` - show what Gradualizer is doing
+    * `--no-fancy` - do not use fancy error messages
+    * `--fmt-location none` - do not display location for easier comparison
+    * `--fmt-location brief` - display location for machine processing
+    * `--fmt-location verbose` - display location for human readers (default)
+
+    * `--no-colors` - do not use colors in printed messages
+    * `--expr-color ansicode` - set color for expressions (default: cyan)
+    * `--type-color ansicode` - set color for types
+    * `--underscore-color ansicode` - set color for the underscored invalid code part
+      in the fancy messages
+
+    # --path_add - add path to the `/ebin` directory for dependencies
```

Comment:
```suggestion
    # --path-add - append a list of comma-delimited paths to the Erlang code path
```

Let's stay consistent with the other options which use a minus.

Reply:
Done